features/step_definitions/owner.steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I visit the owner site$/) do
+  visit '/owner'
+end
+
+When(/^owner adds dish$/) do
+  visit '/owner'
+  fill_in 'name', :with=> 'dish'
+  fill_in 'price', :with=> '34'
+end
+
+Then /^I submit$/ do
+  click_on 'Submit'
+end
```

Comment:
Does this work? Normally this step is `click_link_or_button button` and the text of the button is passed in as a variable: `Given I click the "Submit" button`